packages/components/src/InputFile/InputFile.tsx

```diff
@@ -190,35 +217,41 @@ export function InputFile({
   const dropZone = classnames(styles.dropZoneBase, {
     [styles.dropZone]: variation === "dropzone",
     [styles.active]: isDragActive,
+    [styles.error]: fileRejections?.length > 0,
   });
 
   return (
-    <div
-      {...getRootProps({ className: dropZone })}
-      tabIndex={variation === "button" ? -1 : 0}
-    >
-      <input {...getInputProps()} />
-
-      {variation === "dropzone" && (
-        <Content spacing="small">
-          <Button label={buttonLabel} size="small" type="secondary" />
-          {size === "base" && (
-            <Typography size="small" textColor="textSecondary">
-              {hintText}
-            </Typography>
-          )}
-        </Content>
-      )}
-
-      {variation === "button" && (
-        <Button
-          label={buttonLabel}
-          size={size}
-          type="secondary"
-          fullWidth={true}
-        />
+    <>
+      <div
+        {...getRootProps({ className: dropZone })}
+        tabIndex={variation === "button" ? -1 : 0}
+      >
+        <input {...getInputProps()} />
+
+        {variation === "dropzone" && (
+          <Content spacing="small">
+            <Button label={buttonLabel} size="small" type="secondary" />
+            {size === "base" && (
+              <Typography size="small" textColor="textSecondary">
+                {hintText}
+              </Typography>
+            )}
+          </Content>
+        )}
+
+        {variation === "button" && (
+          <Button
+            label={buttonLabel}
+            size={size}
+            type="secondary"
+            fullWidth={true}
+          />
+        )}
+      </div>
+      {fileRejections?.length > 0 && (
+        <div className={styles.validationErrors}>{validationErrors}</div>
```

Comment:
I was curious what the accessibility UX would be like here with the rejections in their own div

![image](https://github.com/GetJobber/atlantis/assets/11843215/0c9632a6-6eb9-44ce-b8cd-6ea7f805b1bd)

it's not great - you don't get the error message read out to you at all but that's because of the `InputValidation` component itself, not this implementation

Reply:
the screenshot is of the InputValidation component example and not the one you get on `InputFile` to demonstrate that it happens everywhere

Reply:
@ZakaryH how do we do it with InputText? I was pretty sure in that one we actually announce the error when you focus the input (I might be thinking of mobile specifically because I _know_ we solved it there).

Reply:
hmm I'm not 100% sure, I'd need to dig into it more

it seems like we're adding some kind of aria label or something to be read out on the input itself?

![Kapture 2024-01-15 at 11 16 33](https://github.com/GetJobber/atlantis/assets/11843215/fad68a38-7fab-4d49-ab04-da7e4dca96d0)

it's kinda hard to see but after I move off the input without a value, it reads out "you have to enter a name" without me tabbing to the error itself

once I tab to the error it says it's an image with no title and I can't get to the error message